go.mod

```
@@ -1,13 +1,32 @@
 module github.com/pantheon-systems/pauditd
 
 require (
-	github.com/pantheon-systems/certinel v1.2.5
+	github.com/pantheon-systems/certinel v1.2.9
 	github.com/satori/go.uuid v1.2.0
-	github.com/spf13/viper v1.12.0
-	github.com/streadway/handy v0.0.0-20160402200321-f450267a206e
-	github.com/stretchr/objx v0.2.0 // indirect
-	github.com/stretchr/testify v1.7.2
+	github.com/spf13/viper v1.20.1
+	github.com/streadway/handy v0.0.0-20200128134331-0f66f006fb2e
+	github.com/stretchr/testify v1.10.0
 	gopkg.in/alexcesaro/statsd.v2 v2.0.0
 )
 
-go 1.13
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fsnotify/fsnotify v1.8.0 // indirect
+	github.com/go-viper/mapstructure/v2 v2.2.1 // indirect
+	github.com/pelletier/go-toml/v2 v2.2.3 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/sagikazarmark/locafero v0.7.0 // indirect
+	github.com/sourcegraph/conc v0.3.0 // indirect
+	github.com/spf13/afero v1.12.0 // indirect
+	github.com/spf13/cast v1.7.1 // indirect
+	github.com/spf13/pflag v1.0.6 // indirect
+	github.com/subosito/gotenv v1.6.0 // indirect
+	go.uber.org/atomic v1.9.0 // indirect
+	go.uber.org/multierr v1.9.0 // indirect
+	golang.org/x/sys v0.29.0 // indirect
+	golang.org/x/text v0.21.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+go 1.24
```

```suggestion
go 1.24.0
```
This must be a proper Go version (i.e., three components) starting with Go 1.21.0.
https://tip.golang.org/doc/go1.21#tools